tests/utilities/fake_driver.py

```diff
@@ -145,4 +122,8 @@ def __exit__(
         traceback: Optional[TracebackType],
     ) -> None:
         """Exit the session's runtime context."""
-        self.close()
+        if (
+            "initialization_behavior" not in self.options
```

Comment:
initialization_behavior should be a separate constructor parameter like in the MI drivers, not part of the IVI options dictionary.